### PR TITLE
CommitContentSample

### PR DESCRIPTION
**Pull request by gonzalonm**
Ported sample from: https://developer.android.com/samples/CommitContentSampleIME/index.html

### CODE REVIEW

File: android-n/CommitContentSampleIME/CommitContentSampleIME/ImageKeyboard.cs [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+namespace CommitContentSampleIME
+{
+	[Service(Permission = "android.permission.BIND_INPUT_METHOD")]
+	[IntentFilter(new [] { "android.view.InputMethod" })]
+	[MetaData("android.view.im", Resource = "@xml/method")]
+	public class ImageKeyboard : InputMethodService
+	{
+		private const string Tag = "ImageKeyboard";
+		private const string Authority = "com.example.android.supportv13.sampleime.inputcontent";
```
Comment (pjcollins) [inline android-n/CommitContentSampleIME/CommitContentSampleIME/ImageKeyboard.cs]: The upstream sample does have a bug, however we can easily resolve this:

1) For consistency, change the package name of the project - `com.xamarin.commitcontentsampleime`.
2) Change line 39 to the following:
        `const string Authority = "com.xamarin.commitcontentsampleime.inputcontent";`
3) Include the following missing Provider child element to the Application element in AndroidManifest.xml:
```
    <provider android:name="android.support.v4.content.FileProvider" android:authorities="com.xamarin.commitcontentsampleime.inputcontent" android:exported="false" android:grantUriPermissions="true">
        <meta-data android:name="android.support.FILE_PROVIDER_PATHS" android:resource="@xml/file_paths" />
    </provider>
```

**Comment by pjcollins** [comment]
I've suggested a fix for the upstream issue in the sample, and I have some formatting feedback in #162 which applies both this and #165.

Finally, I think it makes sense to include both this and the corresponding App sample into one solution as they're intended to be tested together. What do you think?

File: android-n/CommitContentSampleIME/CommitContentSampleIME/ImageKeyboard.cs (hunk 1 above)
Comment (gonzalonm) [inline android-n/CommitContentSampleIME/CommitContentSampleIME/ImageKeyboard.cs]: Done

**Comment by gonzalonm** [comment]
@pjcollins Yes, I agree with that. Do you want that I add the corresponding App sample into this solution? That way, https://github.com/xamarin/monodroid-samples/pull/165 should be removed and this PR should contain the two projects.

**Comment by gonzalonm** [comment]
@pjcollins I put the two project into this solution. The roundIcon issue was fixed. I removed the "icon" attribute from activity class and I add the roundIcon tag into manifest file and it worked for me. Let me know if you can see it now.

**Comment by pjcollins** [comment]
Excellent, looks good to me!

**Pull request merged**